### DISCUSSION TITLE
how to get This is a ✨special✨ repository containing the organization level discussions for repo-name. Everything posted here will also be visible at the organization level.?

### DISCUSSION THREAD

**Discussion by LevaCompany** [community/community#38381] [Discussions] [answered]
how to get This is a ✨special✨ repository containing the organization level discussions for repo-name. Everything posted here will also be visible at the organization level.?

**Comment by jfullstackdev** [comment, marked as answer]
you mean this Discussion of GitHub Community, 

there are two levels view here, the Org level and the other one the Repo level,

this is the Repo level

https://github.com/community/community/discussions

![image](https://user-images.githubusercontent.com/47092464/200153265-80e946cd-1f89-4767-a372-43e3ff466561.png)

and this is the Org level and I prefer viewing it at the Org level because of the interface

https://github.com/orgs/community/discussions

![image](https://user-images.githubusercontent.com/47092464/200153290-dc5d6659-1d8f-4f6b-975e-2ddd2645d944.png)

see, there is a great difference

**Comment by LevaCompany** [comment]
settings -> discussions -> enable discussions on profile -> select repository
